functional-tests/src/test/java/com/redhat/che/functional/tests/TestTestClass.java

```diff
@@ -25,18 +46,62 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(TestTestClass.class);
 
-  @Inject private TestWorkspace testWorkspace;
-  @Inject private ProjectExplorer projectExplorer;
-  @Inject private Menu menu;
+//  @Inject private TestWorkspace testWorkspace;
+//  @Inject private ProjectExplorer projectExplorer;
+//  @Inject private Menu menu;
 
   @Inject
   @Named("che.host")
   private String cheHost;
 
-  @Inject private DefaultTestUser defaultTestUser;
+  @Inject
+  private DefaultTestUser defaultTestUser;
+
+  @Inject
+  private DefaultHttpJsonRequestFactory defaultHttpJsonRequestFactory;
+
+  @Inject
+  private CheStarterWrapper cheStarterWrapper;
+
+  private String serverBaseURL = "https://" + cheHost + "/";
+  private JsonParser PARSER = new JsonParser();
 
   @Test
-  public void dummyTestCase() {
-    LOG.info("Test is running against:" + cheHost + " with:" + defaultTestUser.getName());
+  public void dummyTestCase() throws Exception {
+    // Get required resources
+    String createWorkspaceRequestJson = "/requests/workspace/osio/create-workspace-request.json";
+    String token = defaultTestUser.obtainAuthToken();
+
+    // Verify che-starter running
+    boolean cheStarterStatus;
+    try {
+      cheStarterStatus = cheStarterWrapper.checkIsRunning();
+    } catch (RuntimeException e) {
+      LOG.error("Che starter communication failed:" + e.getMessage(), e);
+      throw e;
+    }
+    if (!cheStarterStatus) {
+      String errorMsg = "Che starter liveliness probe failed.";
+      LOG.error(errorMsg);
+      throw new RuntimeException("Che starter liveliness probe failed.");
+    }
+
+    // Create workspace
+    String workspaceName;
+    try {
+      workspaceName = cheStarterWrapper.createWorkspace(createWorkspaceRequestJson, token);
+    } catch (Exception e) {
+      LOG.error("Workspace creation failed:"+e.getMessage(),e);
+      throw e;
+    }
+
+//    // Start workspace
```

Comment:
why do we need commented code here ?

Reply:
I want to keep it as a TODO: for the future rebase. I can put it into a separate branch and clean it up from this PR.